### DISCUSSION TITLE
muzei archive does not work

### DISCUSSION THREAD

**Discussion by pihentagy** [romannurik/muzei#779] [Q&A] [answered]
The page https://muzeiapi.appspot.com/archive/2022/12 does not function on Firefox and on Chrome.

(ps: I have an ablocker)

![image](https://user-images.githubusercontent.com/29916/207295639-562358c5-6b3e-45cf-a3c3-5ce8a480e46d.png)

**Comment by romannurik** [comment, marked as answer]
Hm, it seems to work for me right now ... is it still broken? https://muzei.co/archive

**Reply by pihentagy** [reply]
No, it works now.
